Users-Guide.md

```diff
@@ -1,96 +1,106 @@
-map_projection_name attribute provides the name of the map projection.")*
-
-*Include:*
-
-- *Class or attribute name (indicate which it is; capitalize class names according to PDS4 standard)*
-- *PDS4 data type (ASCII_Short_String_Collapsed, ASCII_Real, ASCII_Date, etc.)*
-- *Definition in complete sentences*
-- *Cardinality (minimum and maximum number of values permitted)*
-- *Nillable, yes or no? Explain when it is appropriate to use a nil value*
-- *Minimum and maximum numeric values, if applicable*
-- *Minimum and maximum number of characters, if applicable*
-- *List of valid values, if applicable.*
-
-# Examples
+# Overview of the Mars 2020 Mission Dictionary
+
+The Mars 2020 mission dictionary contains classes and attributes specific to
+the Mars 2020 (Perseverance Rover) mission and its instruments.
+ 
+# How to Include the Mars 2020 Mission Dictionary in a PDS4 Label
+
+The dictionary consists of a set of files with names in the form
+   PDS4_MARS2020_xxxx_yyyy.ext
+   
+where
+   xxxx = the PDS4 Information Model version, e.g. 1G00
+   yyyy = the Mars 2020 Mission Dictionary version, e.g. 1000
+
+and the file extensions are
+   .csv = A comma-separated value table of dictionary attributes
+   .JSON = The dictionary contents in JSON format
+   .sch = The dictionary "rules" as an XML Schematron file
+   .txt = The report generated when the dictionary was built
+   .xml = The PDS4 label that describes this set of files
+   .xsd = The dictionary contents as an XML schema file
+
+Only the schema and Schematron files are needed for validating a PDS4 label.
+
+The latest version of this dictionary may be found on the PDS web site at
+https://pds.nasa.gov/datastandards/dictionaries/index-missions.shtml#mars2020.
+
+The following is an example showing the use of this dictionary in a PDS4 label.
+
+```
+   <?xml version="1.0" encoding="UTF-8"?>
+   <?xml-model href="https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1G00.sch" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+   <?xml-model href="https://pds.nasa.gov/pds4/mission/mars2020/v1/PDS4_MARS2020_1G00_1000.sch" schematypens="http://purl.oclc.org/dsdl/schematron"?>    
+   <Product_Observational xmlns="http://pds.nasa.gov/pds4/pds/v1"
+       xmlns:mars2020="http://pds.nasa.gov/pds4/mission/mars2020/v1"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://pds.nasa.gov/pds4/pds/v1              https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1G00.xsd
+                           http://pds.nasa.gov/pds4/mission/mars2020/v1 https://pds.nasa.gov/pds4/mission/mars2020/v1/PDS4_MARS2020_1G00_1000.xsd">
+```
+
+The namespace for the Mars 2020 Mission Dictionary is http://pds.nasa.gov/pds4/mission/mars2020/v1, abbreviated "mars2020:".
+
+# Dictionary Structure
+
+An outline of the classes and attributes in the dictionary is given below.
+
+```
+    <mars2020:Mars2020_Parameters>
+       <mars2020:Observation_Information>  
+          <mars2020:release_number> 

```

Comment:
Again, I'm hoping we can eventually move to using an _Archive_ LDD that could contain things like `release_number` (as well as other items).  I'd love for GEO and IMG to work together on such a thing.  But, until then, this is cool.